app/src/main/java/com/group80/uoftinder/entities/User.java

```diff
@@ -0,0 +1,126 @@
+package com.group80.uoftinder.entities;
+
+import android.net.Uri;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+
+public class User {
+
+    private final String uid;
+
+    private String userType; // make this final later idk
```

Comment:
See [`Enum` suggestion](https://github.com/CSC207-2022F-UofT/course-project-group-80/pull/25/files#r1027176005) in [Fearure 1 user login](#25).